packages/components/psammead-bulletin/src/index.jsx

```diff
@@ -183,8 +184,10 @@ const TVHeading = styled.h3`
 const RadioHeading = styled.h3`
   ${headingStyles}
   @media(min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
-    padding: ${GEL_SPACING} 0;
+    padding-top: ${GEL_SPACING};
+    padding-bottom: ${GEL_SPACING};
   }
+  ${({ dir }) => (dir === 'ltr' ? `padding-left: 0;` : `padding-right: 0;`)}
```

Comment:
![image](https://user-images.githubusercontent.com/43134742/70619346-c5af3280-1c0c-11ea-9c1f-8345696af4a1.png)
The heading of the radio bulletin should presumably have some left padding?

Reply:
@AlistairGempf Nice spot. Moved it to the media query above